directory.json

```diff
@@ -139,6 +139,7 @@
   "Noklab": "https://cccgoe.de/spaceapi.php",
   "Nottinghack": "https://hms.nottinghack.org.uk/api/spaceapi ",
   "Nova Labs": "http://nova-labs.org/api/",
+  "Odenwilusenz": "spaceapi.odenwilusenz.ch/api.json",
```

Comment:
Can you include either `http://` or `https://` in the URL? I'm not even sure if it would work like this.